+++
title = "An Introduction to Systems Biology by Uri Alon in 2020"
author = ["Cameron Smith"]
lastmod = 2020-09-20T17:33:43-04:00
slug = "An_Introduction_to_Systems_Biology_by_Uri_Alon_in_2020"
draft = false
+++

## [systems biology]({{< relref "systems_biology" >}}), [network inference]({{< relref "network_inference" >}}), [dynamical systems]({{< relref "dynamical_systems" >}}) {#systems-biology--systems-biology-dot-md--network-inference--network-inference-dot-md--dynamical-systems--dynamical-systems-dot-md}


## Part 1. Network motifs {#part-1-dot-network-motifs}


### 1. Transcription networks: basic concepts {#1-dot-transcription-networks-basic-concepts}

<!--list-separator-->

-  1.1 Introduction

<!--list-separator-->

-  1.2 The cognitive problem of the cell

<!--list-separator-->

-  1.3 Elements of transcription networks

    <!--list-separator-->

    -  1.3.1 Separation of timescales

    <!--list-separator-->

    -  1.3.2 The signs on the arrows: activators and repressors

    <!--list-separator-->

    -  1.3.3 The numbers on the arrows: input functions

    <!--list-separator-->

    -  1.3.4 Logic input functions: a simple framework for understanding network dynamics

    <!--list-separator-->

    -  1.3.5 Multi-dimensional input functions govern genes with several inputs

<!--list-separator-->

-  1.4 Dynamics and response time of simple regulation

    <!--list-separator-->

    -  1.4.1 The response time of stable proteins is one cell generation


### 2. Autoregulation: a network motif {#2-dot-autoregulation-a-network-motif}

<!--list-separator-->

-  2.1 Introduction

<!--list-separator-->

-  2.2 Patterns, randomized networks, and network motifs

<!--list-separator-->

-  2.3 Autoregulation is a network motif

<!--list-separator-->

-  2.4 Negative autoregulation speeds the response time of gene circuits

<!--list-separator-->

-  2.5 Negative autoregulation promotes robustness to fluctuations in production rate

<!--list-separator-->

-  2.6 Summary: evolution as an engineer

    <!--list-separator-->

    -  Negative autoregulation is a network motif, a pattern that recurs throughout the network at numbers much higher than expected in random networks. To understand why negative autoregulation is a network motif, we asked what functions it can perform.

    <!--list-separator-->

    -  The second (NAR) design has the advantage that the goal, \\[X\_{st}\\], is reached faster. Furthermore, the fluctuations around \\[X\_{st}\\] due to variations in production and removal rate are reduced in the second, autoregulated design.

    <!--list-separator-->

    -  In an imaginary competition between two species are identical except that one uses circuit A, and the second uses circuit B, the latter would have a selective advantage. Over evolutionary times, structures that have engineering advantages would tend to be selected and appear as network motifs.


### 3. The feedforward loop network motif {#3-dot-the-feedforward-loop-network-motif}

<!--list-separator-->

-  3.1 Introduction

<!--list-separator-->

-  3.2 The feedforward loop is a network motif

<!--list-separator-->

-  3.3 The structure of the feedforward loop gene circuit

<!--list-separator-->

-  3.4 Dynamics of the coherent Type-1 FFL with AND logic

<!--list-separator-->

-  3.5 The C1-FFL is a sign-sensitive delay element

<!--list-separator-->

-  3.6 OR-gate C1-FFL is a sign-sensitive delay for off steps

<!--list-separator-->

-  3.7 The incoherent Type-1 FFL generates pulses of output

<!--list-separator-->

-  3.8 The other six FFL types can also act as filters and pulse generators

<!--list-separator-->

-  3.9 Convergent evolution of FFLs

<!--list-separator-->

-  3.10 Summary


### 4. Temporal programs and the global structure of transcription networks {#4-dot-temporal-programs-and-the-global-structure-of-transcription-networks}

<!--list-separator-->

-  4.1 Introduction

<!--list-separator-->

-  4.2 The single-input module (SIM) network motif

<!--list-separator-->

-  4.3 The SIM can generate temporal gene expression programs

<!--list-separator-->

-  4.4 The multi-output feedforward loop

<!--list-separator-->

-  4.5 The multi-output FFL can generate FIFO temporal programs

<!--list-separator-->

-  4.6 Signal integration by BI-FANS and dense-overlapping regulons

<!--list-separator-->

-  4.7 Network motifs and the global structure of sensory transcription networks

<!--list-separator-->

-  4.8 Interlocked feedforward loops in the <span class="underline"><span class="underline">B. subtilis</span></span> sporulation network


### 5. Positive feedback, bistability, and memory {#5-dot-positive-feedback-bistability-and-memory}

<!--list-separator-->

-  5.1 Network motifs in developmental transcription networks

<!--list-separator-->

-  5.2 Network motifs in protein-protein interaction networks


### 6. How to build a biological oscillator {#6-dot-how-to-build-a-biological-oscillator}


## Part 2. Robustness {#part-2-dot-robustness}


### 7. Kinetic proofreading and conformational proofreading {#7-dot-kinetic-proofreading-and-conformational-proofreading}


### 8. Robust signaling by bifunctional components {#8-dot-robust-signaling-by-bifunctional-components}


### 9. Robustness in bacterial chemotaxis {#9-dot-robustness-in-bacterial-chemotaxis}


### 10. Fold-change detection {#10-dot-fold-change-detection}


### 11. Dynamical compensation and mutant resistance in tissues {#11-dot-dynamical-compensation-and-mutant-resistance-in-tissues}


### 12. Robust spatial patterning in development {#12-dot-robust-spatial-patterning-in-development}


## Part 3. Optimality {#part-3-dot-optimality}


### 13. Optimal gene circuit design {#13-dot-optimal-gene-circuit-design}


### 14. Multi-objective optimality in biology {#14-dot-multi-objective-optimality-in-biology}


### 15. Modularity {#15-dot-modularity}


## Appendices {#appendices}


### A. The input functions of genes: Michaelis-Menten and Hill equations {#a-dot-the-input-functions-of-genes-michaelis-menten-and-hill-equations}


### B. Multi-dimensional input functions {#b-dot-multi-dimensional-input-functions}


### C. Graph properties of transcription networks {#c-dot-graph-properties-of-transcription-networks}


### D. Noise in gene expression {#d-dot-noise-in-gene-expression}
