+++
title = "cancer evolution"
author = ["Cameron Smith"]
lastmod = 2020-09-20T17:36:37-04:00
slug = "cancer_evolution"
draft = false
+++

## See [cevo notes on overleaf](<https://www.overleaf.com/project/5df6860b04ffec00016e465b>) {#see-cevo-notes-on-overleaf--https-www-dot-overleaf-dot-com-project-5df6860b04ffec00016e465b}


## Todo regarding analysis of [stochastic processes]({{< relref "stochastic_processes" >}}) {#todo-regarding-analysis-of-stochastic-processes--stochastic-processes-dot-md}


### Read review [Resolving genetic heterogeneity in cancer by Turajlic, Graham, and Swanton et al 2019]({{< relref "Resolving_genetic_heterogeneity_in_cancer_by_Turajlic,_Graham,_and_Swanton_et_al_2019" >}}) {#read-review-resolving-genetic-heterogeneity-in-cancer-by-turajlic-graham-and-swanton-et-al-2019--resolving-genetic-heterogeneity-in-cancer-by-turajlic-graham-and-swanton-et-al-2019-dot-md}


### Read [Tug-of-war between driver and passenger mutations in cancer and other adaptive processes by McFarland Mirny and Korolev in 2014]({{< relref "Tug-of-war_between_driver_and_passenger_mutations_in_cancer_and_other_adaptive_processes_by_McFarland_Mirny_and_Korolev_in_2014" >}}) @est(2h) @high {#read-tug-of-war-between-driver-and-passenger-mutations-in-cancer-and-other-adaptive-processes-by-mcfarland-mirny-and-korolev-in-2014--tug-of-war-between-driver-and-passenger-mutations-in-cancer-and-other-adaptive-processes-by-mcfarland-mirny-and-korolev-in-2014-dot-md--est--2h--high}


### Read review [Measuring Clonal Evolution in Cancer with Genomics by Williams, Sottoriva, and Graham in 2019]({{< relref "Measuring_Clonal_Evolution_in_Cancer_with_Genomics_by_Williams,_Sottoriva,_and_Graham_in_2019" >}}). @est(2h) @high {#read-review-measuring-clonal-evolution-in-cancer-with-genomics-by-williams-sottoriva-and-graham-in-2019--measuring-clonal-evolution-in-cancer-with-genomics-by-williams-sottoriva-and-graham-in-2019-dot-md--dot-est--2h--high}


### Compare models from [Quantification of subclonal selection in cancer from bulk sequencing data by Williams and Graham in 2018]({{< relref "Quantification_of_subclonal_selection_in_cancer_from_bulk_sequencing_data_by_Williams_and_Graham_in_2018" >}}) and [Tug-of-war between driver and passenger mutations in cancer and other adaptive processes by McFarland Mirny and Korolev in 2014]({{< relref "Tug-of-war_between_driver_and_passenger_mutations_in_cancer_and_other_adaptive_processes_by_McFarland_Mirny_and_Korolev_in_2014" >}}), and Attolini Michor to those in Durrett book @est(6h) @low {#compare-models-from-quantification-of-subclonal-selection-in-cancer-from-bulk-sequencing-data-by-williams-and-graham-in-2018--quantification-of-subclonal-selection-in-cancer-from-bulk-sequencing-data-by-williams-and-graham-in-2018-dot-md--and-tug-of-war-between-driver-and-passenger-mutations-in-cancer-and-other-adaptive-processes-by-mcfarland-mirny-and-korolev-in-2014--tug-of-war-between-driver-and-passenger-mutations-in-cancer-and-other-adaptive-processes-by-mcfarland-mirny-and-korolev-in-2014-dot-md--and-attolini-michor-to-those-in-durrett-book-est--6h--low}


### Run julia code from Williams/Graham papers @est(2h) @high {#run-julia-code-from-williams-graham-papers-est--2h--high}


### Read [Branching process models of cancer by Durrett in 2015]({{< relref "Branching_process_models_of_cancer_by_Durrett_in_2015" >}}) {#read-branching-process-models-of-cancer-by-durrett-in-2015--branching-process-models-of-cancer-by-durrett-in-2015-dot-md}


### Read [Exact solution of a two-type branching process clone size distribution in cell division kinetics by Antal and Krapivsky in 2010]({{< relref "Exact_solution_of_a_two-type_branching_process_clone_size_distribution_in_cell_division_kinetics_by_Antal_and_Krapivsky_in_2010" >}}) {#read-exact-solution-of-a-two-type-branching-process-clone-size-distribution-in-cell-division-kinetics-by-antal-and-krapivsky-in-2010--exact-solution-of-a-two-type-branching-process-clone-size-distribution-in-cell-division-kinetics-by-antal-and-krapivsky-in-2010-dot-md}


### Read review [Tumour heterogeneity and the evolutionary trade-offs of cancer by Hausser and Alon in 2020]({{< relref "Tumour_heterogeneity_and_the_evolutionary_trade-offs_of_cancer_by_Hausser_and_Alon_in_2020" >}}) {#read-review-tumour-heterogeneity-and-the-evolutionary-trade-offs-of-cancer-by-hausser-and-alon-in-2020--tumour-heterogeneity-and-the-evolutionary-trade-offs-of-cancer-by-hausser-and-alon-in-2020-dot-md}


## Todo regarding (cancer) [phylogenetics]({{< relref "phylogenetics" >}}) {#todo-regarding--cancer--phylogenetics--phylogenetics-dot-md}


### Watch Luay Nakhleh lecture on SiCloneFit {#watch-luay-nakhleh-lecture-on-siclonefit}


### [SiCloneFit Bayesian inference of population structure, genotype, and phylogeny of tumor clones from single-cell genome sequencing data by Zafar and Nakhleh in 2019]({{< relref "SiCloneFit_Bayesian_inference_of_population_structure,_genotype,_and_phylogeny_of_tumor_clones_from_single-cell_genome_sequencing_data_by_Zafar_and_Nakhleh_in_2019" >}}) {#siclonefit-bayesian-inference-of-population-structure-genotype-and-phylogeny-of-tumor-clones-from-single-cell-genome-sequencing-data-by-zafar-and-nakhleh-in-2019--siclonefit-bayesian-inference-of-population-structure-genotype-and-phylogeny-of-tumor-clones-from-single-cell-genome-sequencing-data-by-zafar-and-nakhleh-in-2019-dot-md}


### Todo with Ximo {#todo-with-ximo}

<!--list-separator-->

-  Run [pyclone]({{< relref "pyclone" >}}) <https://github.com/aroth85/pyclone>

<!--list-separator-->

-  Study  [pyclone]({{< relref "pyclone" >}})  graphical  model  including  Andrew  Roth’s  lectures <https://www.youtube.com/playlist?list=PLAFFlrzSa6m8PiKPKLBl8Av7RNw4F5-2F>

<!--list-separator-->

-  Run [pyro]({{< relref "pyro" >}}) examples <http://>[pyro]({{< relref "pyro" >}}).ai/examples/•Implement some component of [pyclone]({{< relref "pyclone" >}}) in [pyro]({{< relref "pyro" >}}) and/or pymc3

<!--list-separator-->

-  Study integration of ReMixT with [pyclone]({{< relref "pyclone" >}}) in Interfaces of Malignant and Immunologic ClonalDynamics in Ovarian Cancer by  Zhang and Shah in 2018 including  Shah’s  lecture <https://youtu.be/8cwFV7bWawc>

<!--list-separator-->

-  Run ReMixT for CNV quantification <https://bitbucket.org/dranew/remixt/%E2%80%A2Integrate> ReMixT with [pyclone]({{< relref "pyclone" >}})

<!--list-separator-->

-  Integrate ReMixT output with our own [pyro]({{< relref "pyro" >}})/pymc3 implementation of [pyclone]({{< relref "pyclone" >}})

<!--list-separator-->

-  Complete our own implement of ReMixT and [pyclone]({{< relref "pyclone" >}}) in [pyro]({{< relref "pyro" >}})/pymc3


### [references]({{< relref "references" >}}) {#references--references-dot-md}

<!--list-separator-->

-  Clonal inference with phylogeny

    <!--list-separator-->

    -  Vavoulis, D. V., Cutts, A., Taylor, J. C., & Schuh, A. (2020). A statistical approach for tracking clonal dynamics in cancer using longitudinal next-generation sequencing data. <span class="underline"><span class="underline">BioRxiv</span></span>, 2020.01.20.913236. <https://doi.org/10.1101/2020.01.20.913236>

    <!--list-separator-->

    -  ****Review**** Miura, S., Vu, T., Deng, J., Buturla, T., Oladeinde, O., Choi, J., & Kumar, S. (2020). Power and pitfalls of computational methods for inferring clone phylogenies and mutation orders from bulk sequencing

    <!--list-separator-->

    -  Myers, M. A., Satas, G., & Raphael, B. J. (2019). CALDER: Inferring Phylogenetic Trees from Longitudinal Tumor Samples. <span class="underline"><span class="underline">Cell Systems</span></span>, \_\_8\_\_(6), 514-522.e5. <https://doi.org/10.1016/j.cels.2019.0>

    <!--list-separator-->

    -  El-Kebir, M., Satas, G., & Raphael, B. J. (2018). Inferring parsimonious migration histories for metastatic cancers. <span class="underline"><span class="underline">Nature Genetics</span></span>, \_\_50\_\_(5), 718–726. <https://doi.org/10.1038/s41588-018-0106-z>

    <!--list-separator-->

    -  ****Review**** Schwartz, R., & Schäffer, A. A. (2017, April 1). The evolution of tumour phylogenetics: Principles and practice. <span class="underline"><span class="underline">Nature Reviews Genetics</span></span>. Nature Publishing Group. <https://doi.org/10.1038/nrg.2016.170>

    <!--list-separator-->

    -  Satas, G., & Raphael, B. J. (2017). Tumor phylogeny inference using tree-constrained importance sampling. <span class="underline"><span class="underline">Bioinformatics</span></span>, \_\_33\_\_(14), i152–i160. <https://doi.org/10.1093/bioinformatics/btx270>

    <!--list-separator-->

    -  Deshwar, A. G., Vembu, S., Yung, C. K., Jang, G. H., Stein, L., & Morris, Q. (2015). PhyloWGS: Reconstructing subclonal composition and evolution from whole-genome sequencing of tumors. <span class="underline"><span class="underline">Genome Biology</span></span>, \_\_16\_\_(1), 1–20. <https://doi.org/10.1186/s13059-015-0602-8>

    <!--list-separator-->

    -  Jiao, W., Vembu, S., Deshwar, A. G., Stein, L., & Morris, Q. (2014). Inferring clonal evolution of tumors from single nucleotide somatic mutations. <span class="underline"><span class="underline">BMC Bioinformatics</span></span>, \_\_15\_\_(1), 35. <https://doi.org/10.1186/1471-2105-15-35>

    <!--list-separator-->

    -  Roth, A., Khattra, J., Yap, D., Wan, A., Laks, E., Biele, J., … Shah, S. P. (2014). PyClone: Statistical inference of clonal population structure in cancer. <span class="underline"><span class="underline">Nature Methods</span></span>, \_\_11\_\_(4), 396–398. <https://doi.org/10.1038/nmeth.2883>

<!--list-separator-->

-  Clonal inference without phylogeny

    <!--list-separator-->

    -  [PyClone Statistical inference of clonal population structure in cancer by Roth and Shah in 2014]({{< relref "PyClone_Statistical_inference_of_clonal_population_structure_in_cancer_by_Roth_and_Shah_in_2014" >}}) for inference of clonal population structure in cancer

<!--list-separator-->

-  Phylogenetics proper

    <!--list-separator-->

    -  [Phylogenetic inference using RevBayes by Hohna, Landis, and Heath in 2017]({{< relref "Phylogenetic_inference_using_RevBayes_by_Hohna,_Landis,_and_Heath_in_2017" >}})

<!--list-separator-->

-  Copy number variation

    <!--list-separator-->

    -  [ReMixT Clone-specific genomic structure estimation in cancer by McPherson and Shah 2017]({{< relref "ReMixT_Clone-specific_genomic_structure_estimation_in_cancer_by_McPherson_and_Shah_2017" >}}) for prediction of clone-specific segment copy number and clone specificity of breakpoints
