#+setupfile: ./hugo_setup.org
#+hugo_slug: Phylogenetic_inference_using_RevBayes_by_Hohna,_Landis,_and_Heath_in_2017
#+TITLE: Phylogenetic inference using RevBayes by Hohna, Landis, and Heath in 2017
* [[https://lh3.googleusercontent.com/eZrgzeB0zszjoCjTpiR8SaamsiWHZ92Wr2TrZbb1Rel_yFwDX-ylYxr0WoQf3irtkvyV0SvIJR-6Ko8VmWrrWVqzi09WirJVGhz0aQFfygKGzsuoxTsiqTx7tpp2W9_0Pvt6DgpBcog=w613-h668-no]]
* I. Getting started with RevBayes
** 1. Introduction
** 2. Introduction to RevBayes and Rev
** 3. Reading and manipulating data
* II. Inference
** 4. Introduction to Markov chain Monte Carlo algorithms
* III. Basic phylogeny estimation
** 5. Continuous time Markov model for discrete character evolution
*** 5.1 Overview
**** See [[file:Markov_Processes_for_Stochastic_Modeling_by_Oliver_Ibe_in_2013.org][Markov Processes for Stochastic Modeling by Oliver Ibe in 2013]] section 5.2 for a derivation of the solution to the stationary continuous-time Markov process in terms of the matrix exponential of the instantaneous rate matrix, which is usually named $$Q$$ "5.2 Transient analysis"
**** The substitution models of molecular evolution
***** JC69
***** F81
***** HKY85
***** GTR
***** GTR+Gamma
***** GTR+Gamma+I
*** 5.2 Data and files
*** 5.3 Example: character evolution under the Jukes-Cantor substitution model
**** Steps
***** (1) setting up a Jukes-Cantor (JC) substitution model for an alignment of the cytochrome b subunit; 
***** (2) approximating the posterior probability of the tree topology and node ages (and all other parameters) using MCMC, and; 
***** (3) summarizing the MCMC output by computing the maximum a posteriori tree.
**** [[https://lh3.googleusercontent.com/XsxT1xV-uWFmuy5ifQpYoWLVrjruIFh9FapTQW4GmgaicMz6aoorx7Qb27UEg_SCbUHrIvWqk3OIfcZOKP4bsNe2_JPxsD4gcy_5MWejvhe0vXSmC3TllIjxZ8e4fMt4F2qawtlBox2_gTnqwCiLObpVlvv3uMZ5Ow1VH59ZzXGzL6EZ__2LLxvv3s_DD0nztMnEV6ufOG1LLQCDUVJb5DngFnVwrLSz4n4VspNiuZhboFBXILAEUbbMYfNFl3nrOR0LVEIbycQ7U74fmoOtN_1vSPOQppWW4RqKXi_8ISqrM94YFxkUgI66ziOSnKs3iGnw4I2pbtMdkyI1Q0QV8N-Qr3G1ykeTWvbcBB1x0iO1kQ6B9D6FqlUKfhLGYAO31rNQFNhFv_ROYXIB5aGu3Lwrz4SIH91_1Sz5pH53oSVDvmaSi0cQ7O2G14uNhFuy3lPh5kO49SO9apZkkGcK8Q3pvl4VZNTYiqHLDGw5MLlLPT5DGM8CyLTihryQKEq2ZjQFgXKwR7IGJuG9nVeVv95OiXtDodx8hJoKhu6J9XhS6NjepBcOz1eCrLRyxRwDuzcprk2ijy5-CEQlreTplY5tfsKV78Lrbp1daKwNji7Xhb5P3VHQoyiQH5ORDR1eS_ym-nUQDd5HFbHeDjny9TbhzppJKj9cP_ge9i_4laeIIDI6DPvibWZZdjhWc8ihyfSFA1AYS9WhARa9PFFvc189j4uMRDeiYsARCtZH95TYRBeI=w1018-h475-no]]
**** [[https://lh3.googleusercontent.com/gK5SYbUVLY4h8baOGYBOt5OyLVKYQlm_YKMS_Vl7x5KISoTYVmZpFAfVdED9w1kAzhyyzK6ZhqExV71AKQrpaYqdEfYdEFv0isJ5buKi9iWEI9RunyX4SCnApQ-PRIbM0kaDbcVg0LU=w960-h399-no]]
**** 
**** $$t$$ is the branch length in units of time
*** 5.4 The Hasegawa-Kishino-Yano (HKY) 1985 substitution model
*** 5.5 The General Time-Reversible (GTR) substitution model
*** 5.6 The discrete gamma model of among site rate variation
*** 5.7 Modeling invariable sites
* IV. Model selection, model averaging, and model testing
** 6. 
* V. More on substitution models
** 7. 
* VI. Divergence time estimation
** 8.
** 9.
** 10.
* VII. Diversification rate estimation
** 11.
** 12.
** 13.
** 14.
** 15.
** 16.
* VIII. Gene tree-species tree estimation
** 17.
** 18.
** 19.
* IX. Biogeography
** 20.
** 21.
* X. Phylogenetic comparative method
** 22.
* 
